app/services/kredits.js

```diff
@@ -52,9 +52,10 @@ export default Service.extend({
         });
       }
 
-      function instantiateWithAccount (web3Provider, context) {
+      async function instantiateWithAccount (web3Provider, context) {
         console.debug('[kredits] Using user-provided instance, e.g. from Mist browser or Metamask');
         ethProvider = new ethers.providers.Web3Provider(web3Provider);
+        // const network = await ethProvider.getNetwork();
```

Comment:
we can delete this comment.

Reply:
I left it there intentionally, so that I don't have to look up how to get to the network info of the external provider, and where to do that, when tackling #82.